vocabularies/PersonalData.xml

```diff
@@ -162,12 +162,19 @@ Terms for contact and address information are defined in the [Communication voca
               </Annotation>
             </Record>
             <Record>
-              <Annotation Term="Common.Experimental" />
               <PropertyValue Property="Value" String="LegalEntityID" />
               <Annotation Term="Core.Description" String="The unique identifier of a legal entity" />
               <Annotation Term="Core.LongDescription">
                 <String>A legal entity is a corporation, an association, or any other organization of legal capacity, which has statutory rights and responsibilities.</String>
               </Annotation>
+              <Annotation Term="Core.Revisions">
+                <Collection>
+                  <Record>
+                    <PropertyValue Property="Kind" EnumMember="Core.RevisionKind/Deprecated" />
+                    <PropertyValue Property="Description" String="Deprecated in favor of [`DataControllerID`](#DataControllerID)" />
+                  </Record>
+                </Collection>
+              </Annotation>
```

Comment:
Since that value was experimental, you need not deprecate it but can simply delete it (if you don't want to keep it for reference).

Reply:
I would like to keep it for reference to avoid breaking any unknown integrations using this value...